Sourceforge maitenance has ended.

- https://fosstodon.org/@sourceforgestatus/111389814618523611

We have compile errors now.

```shell
195.6 LD        libavcodec/libavcodec.so.60
196.4 /usr/bin/ld: /usr/local/lib/libfdk-aac.a(genericStds.o): warning: relocation against `stdout@@GLIBC_2.2.5' in read-only section `.text'
196.8 /usr/bin/ld: /usr/local/lib/libvpx.a(vp9_ext_ratectrl.c.o): relocation R_X86_64_PC32 against symbol `stderr@@GLIBC_2.2.5' can not be used when making a shared object; recompile with -fPIC
196.8 /usr/bin/ld: final link failed: bad value
196.8 collect2: error: ld returned 1 exit status
196.8 make: *** [ffbuild/library.mak:119: libavcodec/libavcodec.so.60] Error 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -258,7 +258,7 @@ RUN <<EOF
 EOF
 
 # NVIDIA video codec
-# https://git.videolan.org/?p=ffmpeg/nv-codec-headers
+# https://git.videolan.org/?p=ffmpeg/nv-codec-headers.git
 ARG ENABLE_NVCODEC=0
 ARG NVCODEC_HEADER_VERSION=n12.1.14.0
 RUN <<EOF
@@ -287,7 +287,7 @@ RUN <<EOF
     # https://docs.nvidia.com/video-technologies/video-codec-sdk/12.1/ffmpeg-with-nvidia-gpu/index.html
     NVCODEC_OPTS=""
     if [ "${ENABLE_NVCODEC}" = "1" ]; then
-        NVCODEC_OPTS="--enable-cuda-nvcc --enable-libnpp --extra-cflags=-I/usr/local/cuda/include --extra-ldflags=-L/usr/local/cuda/lib64 --disable-static --enable-shared"
+        NVCODEC_OPTS="--enable-cuda-nvcc --enable-libnpp --extra-cflags=-I/usr/local/cuda/include --extra-ldflags=-L/usr/local/cuda/lib64"
     fi
 
     mkdir -p ${SOURCE_PREFIX}/ffmpeg

